### PR TITLE
Remove deprecated scripts

### PR DESCRIPTION
**Pull request by ddgenome**
Remove atomist-githook script.  Remove TypeScript source for
postInstall.

Update author in package.json.

### CODE REVIEW

File: bin/postInstall.ts [hunk 1]
```diff
@@ -1,24 +0,0 @@
...
-│   🙌 @atomist/cli is now installed.                                      │
-│                                                                          │
-│   Head to the sdm repo (https://github.com/atomist/sdm) for more info.   │
-│                                                                          │
-└──────────────────────────────────────────────────────────────────────────┘`;
-
-// Show an informative and friendly welcome message
-if (!fs.existsSync(userConfigPath())) {
-    process.stdout.write(marked(Banner).trim() + "\n\n");
-}
```
Comment (cdupuis) [inline bin/postInstall.ts]: Why did you delete this file? I kept it so if we need to make future changes we don't have to write straight up JS.

Comment (ddgenome) [reply bin/postInstall.ts]: OK. Shouldn't we commit the js.map and d.ts then too?

Comment (cdupuis) [reply bin/postInstall.ts]: Yeah, to keep it consistent?

**Comment by ddgenome** [comment]
We shouldn't add headers to .d.ts files.

**Comment by ddgenome** [comment]
I've added the other compiled files.

**Pull request merged**